operations/helm/charts/mimir-distributed/templates/lib/topologySpreadConstraints.tpl

```diff
@@ -0,0 +1,12 @@
+{{- define "mimir.lib.topologySpreadConstraints" }}
+{{- $componentSection := include "mimir.componentSectionFromName" . | fromYaml }}
+{{- $topologySpreadConstraintsSection := $componentSection.topologySpreadConstraints }}
+{{- if $topologySpreadConstraintsSection }}
```

Comment:
```suggestion
{{- define "mimir.lib.topologySpreadConstraints" -}}
{{- $componentSection := include "mimir.componentSectionFromName" . | fromYaml -}}
{{- $topologySpreadConstraintsSection := $componentSection.topologySpreadConstraints -}}
{{- if $topologySpreadConstraintsSection -}}
```

part 1/2 remove extra line from generated template. The result of the template gets an extra leading new line from the "nindent" function ,  "n" is for new line.